cibyl/cli/main.py

```diff
@@ -60,6 +86,16 @@ def main():
     # Add arguments from CI & product models to the parser of the app
     for env in orchestrator.environments:
         orchestrator.extend_parser(attributes=env.API)
+    for plugin in plugins:
```

Comment:
As @bregman-arie  mentioned, the openstack models are not connected to the ci models hierarchy. I think one way to do it would be move this loop before the extend_parser call and modify the Job API to add the attributes. Then the extend_parser call in line 88 would already create the right arguments.

Reply:
Thanks for suggestion. I've mixed your idea and mine. Please take a look. :)